tests/reftests/pin.test

```diff
@@ -1057,24 +1070,100 @@ echo "another-inexistant" "inexistant"
 -> installed bar.dev
 Done.
 ### :XI:c: Test opam pin remove <pkg>.<version>
-### opam pin remove --no-action bar.dev
-Ok, bar is no longer pinned to file://${BASEDIR}/bar (version dev)
+### opam switch create removal --empty
+### opam pin ./nip --no-action
+nip is now pinned to file://${BASEDIR}/nip (version ved)
+### opam pin remove --no-action nip.dev
+[ERROR] nip is pinned but not to version dev. Skipping.
+# Return code 2 #
+### opam pin ./nip --no-action
+[NOTE] Package nip is already pinned to file://${BASEDIR}/nip (version ved).
+nip is now pinned to file://${BASEDIR}/nip (version ved)
 ### opam pin remove --no-action nip.wrong-version
 [ERROR] nip is pinned but not to version wrong-version. Skipping.
 # Return code 2 #
+### opam pin remove --no-action nip2.dev
```

Comment:
The version of nip is ved not dev. The test is different from originally intended
```suggestion
### opam pin remove --no-action nip.ved
```

Reply:
cf above